tests/config/test_targetinfo.lua

```diff
@@ -243,6 +243,13 @@
 		test.isequal("bin/Debug/MyProject.xctest/Contents/MacOS", path.getrelative(os.getcwd(), i.bundlepath))
 	end
 
+	function suite.bundlepathSet_onMacSharedLibXCUITest()
```

Comment:
Please add documentation to this test.

Reply:
Can you point me to where pls? Happy to do so.

Reply:
Just look at other unit tests above and add the doc commenr